mempool/mempool.go

```diff
@@ -34,8 +34,10 @@ const (
 
 	// MinHighPriority is the minimum priority value that allows a
 	// transaction to be considered high priority.
+	//
+	// Deprecated: Use mining.MinHighPriority
+	// TODO: Remove in next version update.
 	MinHighPriority = dcrutil.AtomsPerCoin * 144.0 / 250
```

Comment:
Rather than leaving the second definition, which can easily get out of date, please define this in terms of the new one.  That is to say:

```Go
	// MinHighPriority is the minimum priority value that allows a
	// transaction to be considered high priority.
	//
	// Deprecated: Use mining.MinHighPriority
	// TODO: Remove in next version update.
	MinHighPriority = mining.MinHighPriority
```